spec/yardstick/v2_client/grant_status_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Yardstick::V2Client::GrantStatus do
+  let(:gs_client) { Yardstick::V2Client::GrantStatus }
+  let(:token) { 'abssclklkjlkj' }
+
+  describe :find do
+    it 'should populate the token as part of the response handling' do
+      stub_path_response(:get, '/v2/grant_status/100', 200, { id: 100 }, token: token)
+
+      status = gs_client.find(token, 100)
+
+      expect(status.token).to eq(token)
+    end
+  end
+
+  describe :force_terminate do
+    it 'should be able to call the method without passing in the token' do
+      stub_path_response(:put, '/v2/grant_status/100/force_terminate', 200, { id: 100 }, token: token, grant_status: { status: 'veni vidi vici' })
```

Comment:
Line is too long. [146/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.